e2e/test/scenarios/onboarding/auth/signin.cy.spec.js

```diff
@@ -30,40 +30,36 @@ describe("scenarios > auth > signin", () => {
     cy.visit("/");
     cy.findByLabelText("Email address").type(admin.email);
     cy.findByLabelText("Password").type("INVALID" + admin.password);
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Sign in").click();
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.contains("did not match stored password");
+    cy.button("Sign in").click();
+    cy.findByRole("alert")
+      .filter(':contains("did not match stored password")')
+      .should("be.visible");
```

Comment:
`cy.findByRole("alert", {name: ...` does not work; this longer formula is needed.